src/graph.js

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import {Gitgraph} from "@gitgraph/react";
+//import { Gitgraph, Branch } from "@gitgraph/react";
+const gitgraph = require("@gitgraph/react")
+
+let master;
+
+class Graph extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            branches: [],
+            commitMessage:''
+        }
+    }
```

Comment:
I'd recommend you to use the newer React's functional components along with hooks. It's easier and simpler to understand and implement, unlike class based components. There are no `class`es, `this`, `componentDidMount`, etc. while using functional components with hooks, so really easy!